README.md

```diff
@@ -57,6 +57,25 @@ Integer: How often to collect metrics, in minutes. Defaults to `5`.
 
 Integer: How long to retain collect metrics, in days. Defaults to `90`.
 
+##### metrics_server_info
+
+Struct:
+  metrics_server_type => Enum['influxdb','graphite'],
+  hostname            => String,
+  port                => Optional[Integer],
+  db_name             => Optional[String],
+
+Specifies a metrics server to write data to. Currently it supports `influxdb` and `graphite` type servers. The parameters `metrics_server_type` and `hostname` are both required, and `dbname` is required for a `metrics_server_type` of `influxdb`.
```

Comment:
add splunk_hec as data source, link to splunk_hec module since users need that setup first and then just selection splunk_hec, etc.